girderformindlogger/web_client/src/views/body/UserAccountView.js

```diff
@@ -22,6 +22,29 @@ import 'bootstrap/js/tab';
  */
 var UserAccountView = View.extend({
     events: {
+        'submit #g-user-login-form': function (event) {
+            event.preventDefault();
+            this.$('#g-user-login-error-msg').empty();
+
+            var login = this.$('#g-login').val();
+
+            this.user.set(login);
+
+            this.user.off('g:error').on('g:error', function (err) {
+                var msg = err.responseJSON.message;
+                this.$('#g-' + err.responseJSON.field).focus();
+                this.$('#g-user-login-error-msg').text(msg);
+            }, this).off('g:loginChanged')
+                .on('g:loginChanged', function () {
+                    events.trigger('g:alert', {
+                        icon: 'ok',
+                        text: 'login saved.',
+                        type: 'success',
+                        timeout: 4000
+                    });
+                }, this);
+            this.user.updateLogin(this.$('#g-login').val());
+        },
```

Comment:
This is good but unnecessary. This interface is vestigial from the upstream Girder project and not one in use for MindLogger.